src/config.rs

```diff
@@ -16,6 +16,7 @@ use crate::types::PathSource;
 pub const CONFIG_FILE_NAME: &str = "config.toml";
 pub const MAX_CACHE_AGE: Duration = Duration::from_secs(2_592_000); // 30 days
 const DEFAULT_UPDATE_INTERVAL_HOURS: u64 = MAX_CACHE_AGE.as_secs() / 3600; // 30 days
+const ARCHIVE_URL: &str = "https://tldr.sh/assets/tldr.zip";
```

Comment:
Since this is now only used in `default_archive_url`, I think we can just inline it there (and also update it to the new recommended URL, which is https://github.com/tldr-pages/tldr/releases/latest/download/tldr.zip)